src/utils/fs.js

```diff
@@ -27,6 +27,14 @@ export function hasFs() {
   return window._fs_namespace && typeof window[window._fs_namespace] === 'function';
 }
 
+/**
+ * Test if the FullStory recording API is ready to go.  Adds in that getCurrentSessionURL is available to hasFs().
```

Comment:
I just want to clarify this once more - though trying to think about the best wording.
```
Checks if the snippet was able to successfully communicate with FullStory. Either recording has started and a session URL exists or is null if the org is over quota or a domain restriction exists.  
```

Reply:
Put in the comment requested